Session/Conversations/ConversationViewModel.swift

```diff
@@ -197,6 +197,16 @@ public class ConversationViewModel: OWSAudioPlayerDelegate {
                         
                         return SQL("LEFT JOIN \(Profile.self) ON \(profile[.id]) = \(interaction[.authorId])")
                     }()
+                ),
+                PagedData.ObservedChanges(
+                    table: Quote.self,
+                    columns: [.body, .attachmentId],
+                    joinToPagedType: {
+                        let interaction: TypedTableAlias<Interaction> = TypedTableAlias()
+                        let quote: TypedTableAlias<Quote> = TypedTableAlias()
+                        
+                        return SQL("LEFT JOIN \(Quote.self) ON \(quote[.interactionId]) = \(interaction[.id])")
+                    }()
```

Comment:
Do we actually need this change? (since the quote is loading data from the original interaction these values will never get changed)

Reply:
Don't need that anymore. Got it cleaned!